languages/elm/exercises/concept/basics/src/Greet.elm

```diff
@@ -0,0 +1,11 @@
+-- This line 'exports' the `greet` function, so that 
+-- other modules (such as the tests) can use it
+module Greet exposing (greet)
+-- It is possible to export everything with `module Greet exposing (..)`
+
+-- you will need an `import` statement here to import the `toUpper` function from the `String` module
+-- import String
+
+greet : String -> String
+greet personToGreet =
+    Debug.todo "Implement greet"
```

Comment:
As the basics exercise introduces functions, one of its goals is to teach students how to define a function. We've found that the best way to do that is by having the students write the function from scratch. Most tracks "solve" this by having the stub file have some comment on what method to implement: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/lucians-luscious-lasagna/LuciansLusciousLasagna.fs#L3 Note that the basics exercise is rather exceptional, as virtually all other exercises will have stub methods in them (unless there is some inherent benefit of having students define something from scratch, e.g. when introducing the syntax for classes/structs/records).

Reply:
I don't think I agree with this, it just forces a student to copy paste a function from somewhere else. The syntax that a language uses to define a function is not the interesting thing to learn in my opinion. I've made the change anyway though, I just wanted to note my objection.